runtime/src/accounts_db.rs

```diff
@@ -4402,17 +4420,25 @@ impl AccountsDb {
         }
 
         let mut accounts_index_root_stats = AccountsIndexRootsStats::default();
+        let mut not_a_root_count = 0;
+        let mut dead_slots_count = 0;
         let dead_slots: Vec<_> = dead_slots_iter
             .clone()
             .map(|slot| {
+                dead_slots_count += 1;
                 if let Some(latest) = self.accounts_index.clean_dead_slot(*slot) {
                     accounts_index_root_stats = latest;
+                } else {
+                    not_a_root_count += 1;
                 }
                 *slot
             })
             .collect();
         info!("finalize_dead_slot_removal: slots {:?}", dead_slots);
 
+        accounts_index_root_stats.cleaned_roots += dead_slots_count;
```

Comment:
hmm this is called `cleaned_roots`, but `dead_slots_count` also includes unrooted slots, it's incremented outside of the `if let Some(latest) = self.accounts_index.clean_dead_slot(*slot) {` check above. 

How about either:

1) Renaming `cleaned_roots` to `cleaned_slots_count`, or
2)  If we actually want to track the `cleaned_roots`, renaming `dead_slots_count -> dead_roots_count`, and moving the increment inside the `if` check